crates/bevy_ecs/README.md

```diff
@@ -131,7 +131,7 @@ fn main() {
     // Add a Stage to our schedule. Each Stage in a schedule runs all of its systems
     // before moving on to the next Stage
     schedule.add_stage("update", SystemStage::parallel()
-        .with_system(movement)
+        .with_system(movement.system())
```

Comment:
With `0.6` (#2398) the usage of `.system()` is now optional and all examples should no longer contain it.
This change should be reverted.